PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,71 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 190,051 URLs with duplicates  
+Unique list: 187 K
```

Comment:
what do you think about putting the exact number instead of 187k?

Reply:
I am +1 on this

Reply:
Yes sure, done